docs/observability-logging.md

```diff
@@ -0,0 +1,121 @@
+# OpenChoreo Observability
+
+OpenChoreo provides observability for both developers and platform engineers. This is provided through logs as well as metrics in the openchoreo setup. 
+
+## Setting up Observability Logging
+
+ Observability logs are provided through a combination of fluentbit and open search in the data plane. The option of setting up observability is set to false by default. If you need to setup observability in your data plane you can execute the following command based on the type of setup you have chosen.
+
+1. In a single cluster setup
+```
+helm upgrade --install choreo-dataplane oci://ghcr.io/openchoreo/helm-charts/choreo-dataplane \
+            --kube-context kind-choreo \
+            --namespace "choreo-system" --create-namespace 
+            --set certmanager.enabled=false \
+            --set certmanager.crds.enabled=false \
+            --set observability.logging.enabled=true \
+            --version 0.0.0-latest \
+            --timeout 30m
+```
+
+2. In a multicluster setup
+
+> ⚠️ **Important:** This multi-cluster setup (Control plane + Dataplane with FluentBit/OpenSearch) requires minimum 4 CPU and 8GB memory for stable cluster operation.
```

Comment:
shall we use github markdown warnings here? https://github.com/orgs/community/discussions/16925